components/auto-complete/nz-autocomplete-optgroup.component.ts

```diff
@@ -28,9 +27,10 @@ export class NzAutocompleteOptgroupComponent {
   get nzLabel(): string | TemplateRef<void> {
     return this._label;
   }
-
   _label: string | TemplateRef<void>;
 
+  isLabelString: boolean;
```

Comment:
Consider initialize the class fields so that we can turn on `strictPropertyInitialization` in the future to align with angular codebase, see https://github.com/angular/angular/issues/24571#issue-333470063.